test/unit/util-specs.js

```diff
@@ -49,6 +49,8 @@ describe('util.js', function () {
           },
           xpath: '//XCUIElementTypeApplication[@name="🦋"]/XCUIElementTypeWindow',
           path: '0',
+          classChain: '**/XCUIElementTypeWindow',
```

Comment:
It would be good to have more tests. At least one where indexing is present and the second where more that one attribute is included

Reply:
Will add a UT for that

Reply:
@mykola-mokhnach 
Added 2 tests, does it fit what you were expecting